thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/dashboard/resources/JobResource.java

```diff
@@ -77,4 +79,31 @@ public String listTasksForJob(@NotNull @QueryParam("jobId") long jobId) {
     ObjectNode rootNode = JsonResponseUtil.buildResponseJSON(taskDTOs);
     return rootNode.toString();
   }
+
+  @GET
+  @Path("/job/status")
```

Comment:
please provide java doc to discribe the expected input output and the purpose of this endpoint.
How to interpret the results

We are considering to use swagger to mange our API's in the future, so its better to put more comments.